spec/models/manageiq/providers/azure/cloud_manager/provision/cloning_spec.rb

```diff
@@ -113,4 +121,34 @@
       expect(provision.create_nic(false)).to eql(@nic_object.id)
     end
   end
+
+  context "start_clone_task" do
+    before do
+      @resource_group = FactoryBot.create(:azure_resource_group)
+      @dest_name = "test"
+      @clone_options = {clone_options: {name: "test-name", location: "germanycentralwest"}}
```

Comment:
Same with these two if they aren't referenced elsewhere they can be simple local vars in your before block